src/ui/layouts/Delegation.tsx

```diff
@@ -121,7 +121,7 @@ export const DelegateSection = ({ delegates }: { delegates: Delegate[] }) => {
               />
             </div>
           </div>
-          <div className="flex flex-row flex-wrap items-center justify-start gap-y-4 gap-x-7">
+          <div className="grid grid-cols-3 flex-wrap items-center justify-start gap-6">
```

Comment:
Doesn't the grid layout need to be responsive?! currently it breaks the layout on small screens.

Reply:
ah didnt check on smaller screens yet, should be responsive yea, will fix it tomorrow

Reply:
made some additional fixes to make it look decent on mobile. filters for delegates are not perfect, but i guess those will change anyway